plugins/performance-lab/includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -209,9 +209,26 @@ function perflab_aao_get_autoloaded_options_table(): string {
  * @return string HTML formatted table.
  */
 function perflab_aao_get_disabled_autoloaded_options_table(): string {
-	$disabled_options = get_option( 'perflab_aao_disabled_options', array() );
+	$disabled_options = (array) get_option( 'perflab_aao_disabled_options', array() );
 
-	if ( ! is_array( $disabled_options ) ) {
+	$autoload_option_names = array_map(
+		static function ( object $option ): string {
+			return $option->option_name;
+		},
+		perflab_aao_query_autoloaded_options()
+	);
```

Comment:
Another option, 

```suggestion
	$autoload_option_names = wp_list_pluck( perflab_aao_query_autoloaded_options(), 'option_name' );
```

Although the PHPStan typing may not be present.

Reply:
> Although the PHPStan typing may not be present.

Might be something we could contribute to the WP stubs

Reply:
Yeah, the current stub doesn't have any typing beyond what is already in core:

```php
    /**
     * Plucks a certain field out of each object or array in an array.
     *
     * This has the same functionality and prototype of
     * array_column() (PHP 5.5) but also supports objects.
     *
     * @since 3.1.0
     * @since 4.0.0 $index_key parameter added.
     * @since 4.7.0 Uses `WP_List_Util` class.
     *
     * @param array      $input_list List of objects or arrays.
     * @param int|string $field      Field from the object to place instead of the entire object.
     * @param int|string $index_key  Optional. Field from the object to use as keys for the new array.
     *                               Default null.
     * @return array Array of found values. If `$index_key` is set, an array of found values with keys
     *               corresponding to `$index_key`. If `$index_key` is null, array keys from the original
     *               `$input_list` will be preserved in the results.
     */
    function wp_list_pluck($input_list, $field, $index_key = \null)
    {
    }
```

Reply:
Thanks both for feedback. I have reverted the change for `perflab_aao_get_disabled_autoloaded_options_table` so we don't needs to update this. Also PHPSTAN did not show any error if we use `wp_list_pluck` in `hooks.php`. Did i miss anything?